src/Sha1Provider.ts

```diff
@@ -0,0 +1,19 @@
+import * as core from 'webcrypto-core'
+import { sha1 } from '@noble/hashes/sha1'
+import type { HashAlgorithm } from './types'
+
+export class Sha1Provider extends core.ProviderCrypto {
+  public name = 'SHA-1'
+  public usages = []
+
+  public override async onDigest(algorithm: HashAlgorithm, data: ArrayBuffer): Promise<ArrayBuffer> {
+    switch (algorithm.name.toUpperCase()) {
```

Comment:
Can't we do hashes with Askar?

Reply:
Askar does not expose a hashing method over ffi.